eng/ProjectReferences.props

```diff
@@ -131,5 +131,6 @@
     <ProjectReferenceProvider Include="Microsoft.AspNetCore.Components.Browser" ProjectPath="$(RepositoryRoot)src\Components\Browser\src\Microsoft.AspNetCore.Components.Browser.csproj" RefProjectPath="$(RepositoryRoot)src\Components\Browser\ref\Microsoft.AspNetCore.Components.Browser.csproj" />
     <ProjectReferenceProvider Include="Microsoft.AspNetCore.Components" ProjectPath="$(RepositoryRoot)src\Components\Components\src\Microsoft.AspNetCore.Components.csproj" RefProjectPath="$(RepositoryRoot)src\Components\Components\ref\Microsoft.AspNetCore.Components.csproj" />
     <ProjectReferenceProvider Include="Microsoft.AspNetCore.Components.Server" ProjectPath="$(RepositoryRoot)src\Components\Server\src\Microsoft.AspNetCore.Components.Server.csproj" RefProjectPath="$(RepositoryRoot)src\Components\Server\ref\Microsoft.AspNetCore.Components.Server.csproj" />
+    <ProjectReferenceProvider Include="Microsoft.AspNetCore.Analyzers.FeatureDetection.Sources" ProjectPath="$(RepositoryRoot)src\Analyzers\shared\FeatureDetection\Microsoft.AspNetCore.Analyzers.FeatureDetection.Sources.csproj" RefProjectPath="$(RepositoryRoot)src\Analyzers\shared\ref\Microsoft.AspNetCore.Analyzers.FeatureDetection.Sources.csproj" />
```

Comment:
Should I expect to see both projects in here?

Reply:
Also, did I do the source package right? I don't expect this to product a ref....

Reply:
Looks like the .Sources project is still producing a binary, so no..this wasn't done right. This repo doesn't produce any .Sources packages right now, so this will be the first. Recommendations given below.

Reply:
Should I undo changes to this file after fixing that?